server/queue/queue-rabbitmq/src/main/java/org/apache/james/queue/rabbitmq/view/cassandra/configuration/EventsourcingConfigurationManagement.java

```diff
@@ -47,13 +49,21 @@ public EventsourcingConfigurationManagement(EventStore eventStore) {
         this.eventStore = eventStore;
     }
 
-    public Optional<CassandraMailQueueViewConfiguration> load() {
+    public CassandraMailQueueViewConfiguration loadIfAbsent(Supplier<CassandraMailQueueViewConfiguration> initialGenerator) {
+        Preconditions.checkNotNull(initialGenerator);
+
+        return load()
```

Comment:
this logic is wrong. It should be : 
1. I load the configuration from file
2. If no configuration I load the (hardcoded) default one
3. I apply the loaded configuration to the system

it means, for example, that if I start my server without any configuration, I register the default one in the event sourcing system.
It also means that deleting the config file _is_ a change that the system has to handle.
We probably need a way to force the application of a given configuration that would drop the mailqueue entirely.